sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/HPEVideo.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Heading,
+  PageContent,
+  Text,
+  ResponsiveContext,
+  Video,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+
+export const HPEVideo = () => {
+  const breakpoint = useContext(ResponsiveContext);
+
+  return (
+    <PageContent
+      pad={{ vertical: 'xlarge', horizontal: 'xlarge' }}
+      background={{
+        fill: 'horizontal',
+        image: 'url(/video-marketing-background.png)',
+        dark: false,
+        size: 'cover',
+      }}
+      fill="horizontal"
```

Comment:
What does this "fill" prop do on PageContent? Doesn't it already fill?

Reply:
yes had it double thank you!